Sunrise/de.lproj/Localizable.strings

```diff
@@ -175,4 +175,16 @@
 /* Get Directions */
 "Get Directions" = "Anfahrt";
 
+/* Discounts */
+"Discounts" = "Rabatt";
+
+/* Add discount code message */
+"Add your discount code bellow:" = "Fügen Sie Ihren Rabatt Code unten hinzu:";
+
+/* Discount */
+"Discount" = "Rabatt";
+
+/* No discounts */
+"No discounts - please redeem a code." = "Keine Rabatte - bitte erlöschen Sie einen Code.";
```

Comment:
sounds good if you want to use the space and actually tell it.   
If it's the cart details view where you can't edit anything anyways I personally would go for more minimalism and just not mention the code at all if there are none applied.  Since no discounted price is shown, too then there is no need to explain that there is no code applied.